MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -720,17 +731,16 @@ private MediaStream GetMediaStream(bool isAudio, MediaStreamInfo streamInfo, Med
                 stream.AverageFrameRate = GetFrameRate(streamInfo.AverageFrameRate);
                 stream.RealFrameRate = GetFrameRate(streamInfo.RFrameRate);
 
+                bool videoInterlaced = !string.IsNullOrWhiteSpace(streamInfo.FieldOrder)
+                    && !string.Equals(streamInfo.FieldOrder, "progressive", StringComparison.OrdinalIgnoreCase);
+
```

Comment:
Re-ordered this method so that the below comment makes sense as to how `h264MbaffCoded` is formulated